concent_api/core/payments/backends/sci_backend.py

```diff
@@ -23,7 +24,7 @@ class TransactionType(Enum):
 def get_list_of_payments(
     requestor_eth_address:  str,
     provider_eth_address:   str,
-    payment_ts:             int,
+    min_block_timestamp:             int,
```

Comment:
You can fix spaces while being around this code.

Reply:
ok